homework/vector-of-shared-ptrs/vectorFunctions.cpp

```diff
@@ -0,0 +1,33 @@
+#include "vectorFunctions.hpp"
+#include <vector>
+#include <iostream>
+
+std::vector<std::shared_ptr<int>> generate(int count){
+    std::vector<std::shared_ptr<int>> v;
+    for(int i=0;i<count;i++){
```

Comment:
Add space after `if`, `for`, `while` `function` etc...

Add space between operator like `+` `-` `/` `:` `&&` etc..